### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by JakelineWeschenfelder** [community/community#112170] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Estou gostando das badges, porem acho que deveria ser um pouco mais facil conseguir

**Comment by SebastienXia** [comment, marked as answer]
## Hellow, @JakelineWeschenfelder and welcome

> Getting "badges" is not that difficult: you can see all the "badges" via this [link](https://github.com/drknzz/GitHub-Achievements)

**Comment by queenofcorgis** [comment]
Hi @JakelineWeschenfelder , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
